code/object_detector/algorithms/thresh_moment.py

```diff
@@ -0,0 +1,56 @@
+"""
+This algorithm is a manually created one.
+It will:
+    1. Use an HSV representation with a red threshold
+    2. Get moment based on threshold
```

Comment:
I know we explain moment in report, would be nice if there was a small comment here :)